2022/04/shapefiles.md

```diff
@@ -0,0 +1,59 @@
+---
+author: Josh Tolley
+title: "On Shapefiles"
+date: 2022-04-02
+tags:
+- tips
+- open-source
+- tools
+- gis
+- maps
+- postgis
+---
```

Comment:
Do you think it's reasonable to tag this with `postgres` too? That determines whether it'll get syndicated in Planet PostgreSQL.

Reply:
I deliberately left it out, not being sure it was sufficiently postgres-oriented to justify getting picked up by Planet PostgreSQL. But I tend to lean fairly far toward "don't tag it unless I'm really confident", and I wouldn't argue if y'all as Blog Keeper people felt to add the tag.

Reply:
I waffled on this back and forth, but then re-read the Planet PostgreSQL guidelines and looked at the other posts there, and given that all your examples here are about PostGIS, it seems solidly about Postgres.

I also noticed we don't even have a `postgis` tag because we've just used `postgres` + `gis` for that!

So I went with `postgres`.